public/app/features/explore/LiveLogs.tsx

```diff
@@ -23,17 +23,11 @@ const getStyles = (theme: GrafanaTheme) => ({
   logsRowFresh: css`
     label: logs-row-fresh;
     color: ${theme.colors.text};
-    background-color: ${selectThemeVariant(
-      { light: theme.background.logsFresh, dark: theme.background.logsFresh },
-      theme.type
-    )};
+    background-color: ${theme.colors.blueLight}40;
```

Comment:
@ivanahuckova could you pls add comment in code what does the `40` addition to HEX value does? It's not obvious- never heard of it until now 😬

Reply:
I know that you can add alpha to the hex string but I never seen it used tbh. @dprokop I usually used some js lib for color manipulation but as we already have d3 we could use https://github.com/d3/d3-color/blob/v1.3.0/README.md#color for that having:
```
background-color: ${d3.color(theme.colors.blueLight).opacity(0.25)};
```
which probably is more expressive (though the parsing may have some perf impact so maybe it makes sense to put the modified colors outside of the function)

Reply:
@dprokop I've added comment. Do you think it is clear now? I was also checking if emotion doesn't have hex to rgb built in converter, but I couldn't find anything. 😕